### ISSUE TITLE
Data set generation

### ISSUE THREAD

**Issue by KarthikaKP** [PeterChe1990/GRU-D#1]
Hi,
I am new in to machine learning.Can you please provide the code for generating the data required by GRU-D from PhysioNet Challenge 2012 dataset  for predicting 4 tasks together.

Thanks in advance,
karthika

**Comment by PeterChe1990** [comment]
Hi Karthika,

We use only `Training set A` from PhysioNet 2012 Challenge Dataset.

For the labels of 4 tasks:
In-hospital mortality: `In-hospital_death == 1` in `Outcomes-a.txt`
Length-of-stay less than 3 days: `Length_of_stay < 3` in `Outcomes-a.txt`
Whether the patient had a cardiac condition: `ICUType <= 2` or not in `$PID.txt`
Whether the patient was recovering from surgery: `(ICUType mod 2) == 0` or not in `$PID.txt`

Please refer to the paper and the supplementary S3.2 for more details. Currently, we do not have cleaned iPython notebook scripts to share.

**Comment by KarthikaKP** [comment]
Thanks a lot for your reply.Will look in to it and mail you if I have any doubt.

Thanks,
Karthika
